test/Microsoft.ML.Predictor.Tests/TestGamPublicInterfaces.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Trainers.FastTree;
+using Xunit;
+
+namespace Microsoft.ML.RunTests
+{
+    public class TestGamPublicInterfaces
+    {
+        public TestGamPublicInterfaces()
+        {
+        }
+
+        [Fact]
+        [TestCategory("FastTree")]
```

Comment:
>FastTree [](start = 23, length = 8)

should there be a gam category?

Reply:
GAM uses the FastTree internals, so we keep it in `FastTree`. 🤷‍♂️